kivy/uix/behaviors/resize.py

```diff
@@ -0,0 +1,383 @@
+    defaults to 0.5 centimeters.
+    '''
+
+    resizable_left = BooleanProperty(False)
+    '''Enable / disable resizing on left side
+
+    :attr:`resizable_left` is a :class:`~kivy.properties.BooleanProperty` and
+    defaults to False.
+    '''
+
+    resizable_right = BooleanProperty(False)
+    '''Enable / disable resizing on right side
+
+    :attr:`resizable_right` is a :class:`~kivy.properties.BooleanProperty` and
+    defaults to False.
+    '''
+
+    resizable_up = BooleanProperty(False)
+    '''Enable / disable resizing on upper side
+
+    :attr:`resizable_up` is a :class:`~kivy.properties.BooleanProperty` and
+    defaults to False.
+    '''
+
+    resizable_down = BooleanProperty(False)
+    '''Enable / disable resizing on lower side
+
+    :attr:`resizable_down` is a :class:`~kivy.properties.BooleanProperty` and
+    defaults to False.
+    '''
+
+    resizing_left = BooleanProperty(False)
+    '''A State which is enabled/disabled depending on the position relative to the left resize border
+    It is switched to True when mouse is inside the left resize border and False when it isn't.
+    It adjusts the mouse cursor and manages resizing when touch is moved.
+
+    :attr:`resizing_left` is a :class:`~kivy.properties.BooleanProperty` and
+    defaults to False.
+    '''
+
+    resizing_right = BooleanProperty(False)
+    '''A State which is enabled/disabled depending on the position relative to the right resize border
+    It is switched to True when mouse is inside the right resize border and False when it isn't.
+    It adjusts the mouse cursor and manages resizing when touch is moved.
+
+    :attr:`resizing_right` is a :class:`~kivy.properties.BooleanProperty` and
+    defaults to False.
+    '''
+
+    resizing_up = BooleanProperty(False)
+    '''A State which is enabled/disabled depending on the position relative to the upper resize border
+    It is switched to True when mouse is inside the upper resize border and False when it isn't.
+    It adjusts the mouse cursor and manages resizing when touch is moved.
+
+    :attr:`resizing_up` is a :class:`~kivy.properties.BooleanProperty` and
+    defaults to False.
+    '''
+
+    resizing_down = BooleanProperty(False)
+    '''A State which is enabled/disabled depending on the position relative to the lower resize border
+    It is switched to True when mouse is inside the lower resize border and False when it isn't.
+    It adjusts the mouse cursor and manages resizing when touch is moved.
+
+    :attr:`resizing_down` is a :class:`~kivy.properties.BooleanProperty` and
+    defaults to False.
+    '''
+
+    resizing = BooleanProperty(False)
+    '''State of widget resizing.
+    It is switched to True when a resize border is touched and back to False when it is released.
+    It manages resizing when touch is moved.
+
+    :attr:`resizing` is a :class:`~kivy.properties.BooleanProperty` and
+    defaults to False.
+    '''
+
+    dont_move = BooleanProperty(False)
```

Comment:
I aggree, short variable names are good, but I also like to describe what I am doing.I am not sure what to name it.
But i would rename it to resize_move, if another dev aggrees that it's a good name.